src/proj10BittingCerratoCohenEllmer/bantam/parser/Parser.java

```diff
@@ -68,7 +68,9 @@ private void advanceIfTokenMatches(Token.Kind tokenKindExpected) {
         }
         if (currentToken.kind == tokenKindExpected) {
             advance(); // move on to the next token
-        } else {
+        } else if (currentToken.kind == VAR && tokenKindExpected == IDENTIFIER){
```

Comment:
Can you explain a little bit more why? Beyond just saying "we are okay"

Reply:
check out my updated comment